emsdk.py

```diff
@@ -2775,12 +2784,10 @@ def expand_sdk_name(name, activating):
     name = name.replace('upstream-master', 'upstream-main')
   if name in ('latest-fastcomp', 'latest-releases-fastcomp', 'tot-fastcomp', 'sdk-nightly-latest'):
     exit_with_fastcomp_error()
-  if name in ('latest', 'sdk-latest', 'latest-64bit', 'sdk-latest-64bit'):
-    # This is effectly the default SDK
-    return str(find_latest_releases_sdk('upstream'))
-  elif name in ('latest-upstream', 'latest-clang-upstream', 'latest-releases-upstream'):
-    return str(find_latest_releases_sdk('upstream'))
-  elif name in ('tot', 'sdk-tot', 'tot-upstream'):
+  if name in ('latest', 'sdk-latest', 'latest-64bit', 'sdk-latest-64bit', 'latest-upstream', 'latest-clang-upstream', 'latest-releases-upstream'):
```

Comment:
I wonder if this could be part of the new `aliases` mechanism too - that is, allow alias to point to other aliases and make function resolve recursively.

The depth would always be only 1 or 2, but it would allow to encode `"sdk-latest": "latest"` etc. as regular alises in the same config.

Or maybe I'm overcomplicating things?

Reply:
I thought of that too.. but for some reason I found that to be ugly... I'll revisit.

Reply:
Done